### PR TITLE
rules/klayout/lvs: try multiple netlist formats

### PR DESCRIPTION
**Pull request by proppy**
Related: https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/pull/99 https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/issues/101 https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/issues/39 https://github.com/RTimothyEdwards/open_pdks/issues/346

This allow to discover LVS netlists generated by xschem with their default extension (`.spice`).

### CODE REVIEW

File: rules/klayout/lvs/gf180mcu.lvs [hunk 1]
```diff
@@ -255,13 +255,15 @@ if $schematic
...
 else
-  begin  
-    schematic("#{source.cell_name}.cdl", reader)
-    logger.info("Netlist file: #{source.cell_name}.cdl")
-  rescue 
-    puts "No schematic loaded , please add your netlist file"
-    logger.error("No schematic loaded , please add your netlist file")
-  end 
+  exts = ["ngspice", "spice", "cdl"]
+  candidates = exts.map{|ext| "#{source.cell_name}.#{ext}"}
```
Comment (atorkmabrains) [inline rules/klayout/lvs/gf180mcu.lvs:259]: @proppy Could you please remove 'ngspice' as we can't use the netlist used for simulation for LVS?

**Review by atorkmabrains — CHANGES_REQUESTED** [review]

File: rules/klayout/lvs/gf180mcu.lvs (hunk 1 above)
Comment (atorkmabrains) [reply rules/klayout/lvs/gf180mcu.lvs:259]: @proppy ?

**Comment by proppy** [comment]
PTAL

**Review by atorkmabrains — APPROVED** [review]
Thanks for the update.

**Pull request merged**